sdk/nodejs/runtime/closure/createClosure.ts

```diff
@@ -1058,7 +1058,7 @@ function getOrCreateEntry(
     }
 
     function captureModule(moduleName: string) {
-        if (moduleName[0] === ".") {
+        if (moduleName.startsWith(".") && !moduleName.startsWith("./node_modules/")) {
```

Comment:
Just for future sanity - we may want to add a comment to `findModuleName` with examples of the paths it returns.  The paths stored in the Node module cache are not what you might naively expect, so interpreting these correctly is easy to get wrong.

Reply:
sounds good.

Reply:
Done.